ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionViewModel.kt

```diff
@@ -34,21 +35,23 @@ import io.reactivex.processors.BehaviorProcessor
 import io.reactivex.processors.FlowableProcessor
 import io.reactivex.processors.PublishProcessor
 import javax.inject.Inject
+import javax.inject.Provider
 
 /** View model for the Data Collection fragment. */
 class DataCollectionViewModel
 @Inject
 internal constructor(
   private val submissionRepository: SubmissionRepository,
   private val locationOfInterestHelper: LocationOfInterestHelper,
+  private val popups: Provider<EphemeralPopups>,
 ) : AbstractViewModel() {
   val submission: @Hot(replays = true) LiveData<Loadable<Submission>>
   val jobName: @Hot(replays = true) LiveData<String>
   val loiName: @Hot(replays = true) LiveData<String>
   /** "Continue" button clicks. */
   private val continueClicks: @Hot PublishProcessor<Nil> = PublishProcessor.create()
   /** Outcome of user clicking "Continue". */
-  val continueResults: Observable<String?>
+  private val continueResults: Observable<String?>
```

Comment:
Should we get rid of this observable as well? `onContinueClicked()` is already getting called from the layout directly. Not sure what's the purpose of this is now